### PR TITLE
Add image alt and creater Twitter handle to Twitter Cards

### PR DESCRIPTION
**Pull request by BasThomas**
Improvements on #278.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -6,14 +6,17 @@
     <meta name="author" content="{{ site.curator.name }}">
 
     <meta name="twitter:card" content="summary" />
-    <meta name="twitter:site" content="@swiftlybrief" />
-    <meta name="twitter:image" content="https://swiftweekly.github.io/img/logo.png" />
+    <meta name="twitter:site" content="{{ site.twitter.username }}" />
```
Comment (BasThomas) [inline _includes/head.html]: This line does now remove the `@`, but I think that's fine; Twitter should be able to handle that, I hope! 🙈

Comment (gregheo) [inline _includes/head.html]: can always `@{{ site.twitter.username }}`, right? Or use `prepend`? Not sure how strict the Twitter is about this but we'll find out ;)

Comment (BasThomas) [inline _includes/head.html]: Yep, `@{{}}` would work fine :)

**Comment by jessesquires** [comment]
nice! 💯 🥇 

thanks guys! 👌

**Pull request merged**